tests/acceptance/features/recursion_error.feature

```diff
@@ -0,0 +1,1090 @@
+Feature: Checking Django failure
```

Comment:
Please, make this more explicit. It's not entirely self-explanatory.

Think: "What type of failure?"